src/components/alert/bl-alert.ts

```diff
@@ -0,0 +1,139 @@
+import { CSSResultGroup, html, LitElement, TemplateResult } from 'lit';
+import { customElement, property, state } from 'lit/decorators.js';
+import { event, EventDispatcher } from '../../utilities/event';
+import style from './bl-alert.css';
+import '../icon/bl-icon';
+import { ifDefined } from 'lit/directives/if-defined.js';
+import { iconConverter } from './icon.converter';
+import { ButtonVariant, ButtonKind, ButtonSize } from '../button/bl-button';
+import { classMap } from 'lit/directives/class-map.js';
+
+export type AlertVariant = 'info' | 'warning' | 'success' | 'error';
+
+/**
+ * @tag bl-alert
+ * @summary Baklava Alert component
+ *
+ * @cssproperty --bl-alert-display - Sets the display property of button. Default value is 'inline-block'.
```

Comment:
I'm not sure if we need a variable like this. If this only sets the `display` of `:host` then it can be already set outside (host element is on the outside of the shadowdom and styles of the host element can be overriden by component user).

Reply:
I've realized that i kinda misunderstood the concept of `:host`.
In current state of this component when `closable` button clicked component adds class `hidden` to main wrapper (which already has `alert` class) and sets `display: none;` to main wrapper, not the `host`.
It's better to set display of `host` to `none` as you've said but i couldn't manage to change hosts `display` value without using a reflected attribute (`:host[(hidden)]`). 
I've override `--bl-display` of `host` when component has `class="alert hidden"` but it seems like not working either.
I'm feeling like i'm missing the obvious 😄, would you mind helping me with an example for close button to change `host`'s visibility?

Reply:
I think it make sense to have a reflected attribute named `hidden` (or maybe `closed` is better to be aligned with `closable` attribute) and style it as 

```css
:host([closed]) {
   display: none;
}
```

With this way we can animate showing/hiding alert in the feature.

You can also add public `open()` and `close()` methods just sets the `closed` property accordingly. Our internal close button also can just call same `close()` method.

Does this make sense?

Reply:
Public methods can be useful for the consumer to programmatically show/hide an alert. Of course `closed` will be `false` by default, so `open()` only help once `closed` is `true`.

Reply:
Ok this makes sense, i'm refactoring component accordingly. Thank you!

Reply:
should public methods mentioned here be added to issue page? @muratcorlu

Reply:
Yes, public methods also should be documented in the issue and the storybook.